tests/common/factories.py

```diff
@@ -143,3 +143,13 @@ class Meta:  # pylint: disable=no-init, old-style-class
 
     key = factory.LazyAttribute(lambda _: FAKER.domain_word())
     value = factory.LazyAttribute(lambda _: FAKER.catch_phrase())
+
+
+class Flag(ModelFactory):
+
+    class Meta:  # pylint: disable=no-init, old-style-class
+        model = models.Flag
+        force_flush = True
```

Comment:
Not really, we seem to have a bit of a mess of how we handle model factories. Most of the model factories set `force_flush`, except for `User` and `AuthTicket`. So I went with the majority and set it.